test/fixtures.ts

```
@@ -39,3 +39,11 @@ class MyClass {
 const obj2 = {
   method: () => {}
 }
+
+export function exportFunction() { return 1; } // expect-error yenz/export-at-end-of-file // fix: function exportFunction() { return 1; }
```

ESLint

[error] Declare this without inline export and list it in a single export statement at the end of the file.

(yenz/export-at-end-of-file)